src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsFields.hpp

```diff
@@ -46,6 +46,10 @@ using array2dLayoutStrain = array2d< real64, cells::STRAIN_PERM >;
 using arrayView2dLayoutStrain = arrayView2d< real64, cells::STRAIN_USD >;
 using arrayViewConst2dLayoutStrain = arrayView2d< real64 const, cells::STRAIN_USD >;
 
+using array2dLayoutAvgStress = array2d< real64, cells::STRAIN_PERM >;
+using arrayView2dLayoutAvgStress = arrayView2d< real64, cells::STRAIN_USD >;
+using arrayViewConst2dLayoutAvgStress = arrayView2d< real64 const, cells::STRAIN_USD >;
```

Comment:
can we call `cells::STRAIN_PERM` something like`cells::TENSOR_TWO_PERM`?

Reply:
`SYMMETRIC_RANK_TWO_TENSOR`

Reply:
the only issue of making it too long is that every time we want to get a view out then we have to write like:
`arrayView2d< real64 , cells::SYMMETRIC_RANK_TWO_TENSOR_PERM >` but that's indeed the best name. Although I don't think the permutation would change for non symmetric tensor.

Reply:
Yes, you are right `RANK_TWO_TENSOR_PERM` should be enough.

Reply:
It's long, but clear and general.